app/assets/javascripts/application.js

```diff
@@ -19,3 +19,7 @@
 //= require tinymce-jquery
 //= require turbolinks
 //= require_tree .
+
+function myFunction(x) {
```

Comment:
We need to call this something other than `myFunction` that tells us what this does so we can maintain this.

Reply:
right